### ISSUE TITLE
Add Welcome APIs

### ISSUE THREAD

**Issue by cbornet** [cbornet/netatmo-swagger-decl#4]

**Comment by wep4you** [comment]
Hello i try to implement the netatmo welcome camera into Openhab2. Because the Swagger API for welcome is outdated clinique pointed me to your repository. If you need ill help in the implementation, just give me a hint or starting point what to do.

**Comment by cbornet** [comment]
Great ! I don't have the Welcome camera so any help is much appreciated !
To start, you can open the swagger spec (https://raw.githubusercontent.com/cbornet/netatmo-swagger-decl/master/swagger.json) in [swagger-editor](http://editor.swagger.io) then use the documentation from netatmo developper site to add the APIs.
To validate the spec, I generally use https://github.com/cbornet/netatmo-swagger-api java-jersey2 codegen and configure jackson with 
```java
mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true);
```
in JSON.java (https://github.com/cbornet/netatmo-swagger-api/blob/master/samples/netatmo/client/java-jersey2/src/main/java/io/swagger/client/JSON.java) so that it fails if I have a field missing in the response model.

Then the best is to pull a PR to the repo and I will merge.

**Comment by wep4you** [comment]
Ok, ill have a look at the informations and try to get all up and running, in the meantime i also forked your git repository, to send a PR when im done

**Comment by wep4you** [comment]
@cbornet Editor is ok, but i can't try the changes, i generated a key but get the errors when i want to try the apis:
´´´
 This is a cross-origin call. Make sure the server at api.netatmo.net accepts GET requests from editor.swagger.io
´´´

I cloned the netatmo-swagger-api for  code validation and changed the settings in JSON.java
What i don't know is where to put my changed son file from the step before. If i just try to run  java-jersey from command line, a swagger...jar was download the first time of execution, but i get the following error: 
´´´
************
generating java-jersey2
************
Exception in thread "main" io.airlift.airline.ParseArgumentsUnexpectedException: Found unexpected parameters: [--library, jersey2, --artifact-id=-java-jersey2]
	at io.airlift.airline.Cli.validate(Cli.java:148)
	at io.airlift.airline.Cli.parse(Cli.java:116)
	at io.airlift.airline.Cli.parse(Cli.java:97)
	at io.swagger.codegen.SwaggerCodegen.main(SwaggerCodegen.java:35)
´´´

Hope you could help me out, don't know how to test my changes and generate the java classes to get going with welcome implementation in OH2. Thanks

**Comment by cbornet** [comment]
Strange : normally GET requests should work cross-domain. There is a problem with POST requests because of http://forum.netatmo.com/viewtopic.php?f=5&t=4908&p=20638&hilit=cors#p20638
As for the jersey2 generator, I'll have a look.

**Comment by wep4you** [comment]
Hello, any news on this @cbornet ?
Ill try to get this running for the Openhab2 netatmo binding

**Comment by cbornet** [comment]
Hello, sorry for the wait. I forgot about this issue... I'll go back on this asap.

**Comment by cbornet** [comment]
Fixed by #11
